src/actions/MetadataActions.js

```diff
@@ -43,6 +48,9 @@ class MetadataActions {
     return secretin
       .deleteSecret(secret.id)
       .catch((error) => {
+        this.deleteSecretFailure({
+          currentUser: secretin.currentUser,
+        });
```

Comment:
As I now know we can easily have metadata, please return only those. Current user is useless here :)
ℹ️ Also for the success action.